gradle/libs.versions.toml

```diff
@@ -0,0 +1,46 @@
+[versions]
+gradleVersion = "8.1.4"
+kotlinVersion = "1.9.20"
+
+dokkaVersion = "1.9.10"
+appcompatVersion = "1.6.1"
+recyclerviewVersion = "1.3.2"
+viewPager2Version = "1.0.0"
+swiperefreshlayoutVersion = "1.1.0"
+annotationVersion = "1.2.0"
+testRunnerVersion = "1.5.2"
+testRulesVersion = "1.5.0"
+
+espressoVersion = "3.5.1"
+
+junitVersion = "4.13.2"
+junitExtVersion = "1.1.5"
+multidexVersion = "2.0.1"
+materialVersion = "1.10.0"
+
+detekt = "1.17.1"
+
+[libraries]
+gradle = { module = "com.android.tools.build:gradle", version.ref = "gradleVersion" }
```

Comment:
I would name it `agp` or `androidGradle`. Gradle is too broad, I'am reading

```
dependencies {
     classpath(libs.gradle)
}
```

from previous file and have no idea what is it.